indico/web/templates/forms/linking_widget.html

```diff
@@ -5,7 +5,8 @@
 {% block html %}
     <div id="{{ field.id }}">
         {% for subfield in field %}
-            <div class="i-radio i-linking form-subfield{% if has_error[subfield.data] %} has-error{% endif %}">
+            <div class="i-radio i-linking form-subfield flexrow f-a-center
+                {%- if has_error[subfield.data] %} has-error{% endif %}">
```

Comment:
I'd change this to `{{ 'has-error' if has_error[subfield.data] }}` (and not bother about the whitespace)